hazelcast/src/main/java/com/hazelcast/config/QueueConfig.java

```diff
@@ -19,6 +19,8 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import static com.hazelcast.config.CacheSimpleConfig.MIN_BACKUP_COUNT;
```

Comment:
instead of using CacheSimpleConfig static variable, can you create new static variable in this class?